site/src/js/index.js

```diff
@@ -46,10 +46,10 @@ const renderSwag = swag => {
         })
         .sort((a, b) => {
             switch (sorting) {
-                case 'Alphabetical':            return a.name.toLowerCase() > b.name.toLowerCase();
-                case 'Alphabetical, reversed':  return a.name.toLowerCase() < b.name.toLowerCase();
-                case 'Difficulty':              return difficultyIndex(a.difficulty) > difficultyIndex(b.difficulty);
-                case 'Difficulty, reversed':    return difficultyIndex(a.difficulty) < difficultyIndex(b.difficulty);
+            case 'Alphabetical':            return a.name.toLowerCase() > b.name.toLowerCase();
```

Comment:
would suggest not reformatting like this, as it seems unnecessary to the PR

Reply:
I'm guessing the linter is doing that.

Reply:
Looking at the whitespace removal, i think you're right. Still shouldn't commit unnecessary changes like this, unless these changes were causing linter errors, that is.

Reply:
@plibither8 The linter (which you had put) is universal, right? Do you know why it is behaving differently for him?

Reply:
I had forgot to run the linter before commiting the file :sweat_smile:

Reply:
Haha, np.

Reply:
Yes, i'm using atom with `linter-eslint` on save. It follows the local `.eslintrc` file so it should respect the rules.

https://eslint.org/docs/rules/indent

Here we can see : `"SwitchCase" (default: 0) enforces indentation level for case clauses in switch statements`
And: `Indent of 2 spaces with SwitchCase set to 0 will not indent case clauses with respect to switch statements.`

Althrough I see Ident configured on 4 spaces in the `.eslintrc`, I dont know why it's not indenting the `case` statements.